components/slider/style/index.less

```
@@ -66,6 +66,15 @@
     &.@{ant-prefix}-tooltip-open {
       border-color: @slider-handle-color-tooltip-open;
     }
+
+    &::after {
+      position: absolute;
+      top: -10px;
+      right: -10px;
+      bottom: -10px;
+      left: -10px;
```

```suggestion
      left: -6px;
```

6px seems to be reasonable.